src/components/CreateVmWizard/steps/SummaryReview.js

```diff
@@ -290,13 +291,13 @@ const SummaryReview = ({
             {msg.createVmWizardReviewSuccess()}
           </div>
           { progress.messages && progress.messages.length > 0 && (
-            <Alert type='success' >
-              {progress.messages.map((message, index) => (
-                <div key={`message-${index}`}>
-                  { message }
-                </div>
-              )
-              )}
+            <Alert variant='success' isInline title={msg.messages()}>{
```

Comment:
"Messages" is ok here for the title, but seems a bit awkward.  I have no better suggestion.  At least it is not likely to be displayed!